app/services/case_filter/case_multi_choices_filter_base.rb

```diff
@@ -6,7 +6,7 @@ def identify
         if filter_attributes.empty?
           raise '#dentify should be defined in sub-class of CaseMultiChoicesFilterBase'
         else
-          filter_attributes[0]
+          filter_attributes[0].to_s
         end
       end
 
```

Comment:
Tiny typo here. "#dentify" instead of "#identify" :) 

there's a Ruby style thing about guard clauses (see https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/GuardClause) - 

Wondering is the message for the exception right? I'm probably missing something but it looks like maybe it should say something like 

```suggestion
def identify
  raise 'filter_attributes should be defined in sub-class of CaseMultiChoicesFilterBase' if filter_attributes.empty?
  filter_attributes.first.to_s
end
```

Reply:
haven't got used to 'ruby' feature --> offering the 'first' , 'second' which is quite useful